feature-libs/checkout/components/services/express-checkout.service.ts

```diff
@@ -91,10 +92,12 @@ export class ExpressCheckoutService {
         }),
         filter(([, addressesLoadedSuccess]) => addressesLoadedSuccess),
         take(1),
-        switchMap(([addresses]) => {
+        switchMap(([addresses, , deliveryAddress]) => {
           const defaultAddress =
             addresses.find((address) => address.defaultAddress) || addresses[0];
-          if (defaultAddress && Object.keys(defaultAddress).length) {
+          if (deliveryAddress && Object.keys(deliveryAddress).length > 0) {
```

Comment:
Is there a case when the `deliveryAddress` is defined without any properties?

Reply:
Yes. The store currently returns {} instead of undefined/null for empty address